app/dao/notifications_dao.py

```diff
@@ -438,17 +434,14 @@ def _timeout_notifications(current_statuses, new_status, timeout_start, updated_
         Notification.status.in_(current_statuses),
         Notification.notification_type != LETTER_TYPE
     ).all()
-    for table in [NotificationHistory, Notification]:
-        q = table.query.filter(
-            table.created_at < timeout_start,
-            table.status.in_(current_statuses),
-            table.notification_type != LETTER_TYPE
-        )
-        q.update(
-            {'status': new_status, 'updated_at': updated_at},
-            synchronize_session=False
-        )
-    # return a list of q = notification_ids in Notification table for sending delivery receipts
+    Notification.query.filter(
+        Notification.created_at < timeout_start,
+        Notification.status.in_(current_statuses),
+        Notification.notification_type != LETTER_TYPE
+    ).update(
+        {'status': new_status, 'updated_at': updated_at},
+        synchronize_session=False
+    )
```

Comment:
not one for right now cos this change is big enough, but in the future we can use `update ... returning ...` to only execute the query once instead of twice

https://docs.sqlalchemy.org/en/13/dialects/postgresql.html#insert-update-returning